pom.xml

```diff
@@ -364,10 +364,61 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>${license-maven.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.0.0</version>
+                </plugin>
             </plugins>
         </pluginManagement>
 
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <fail>true</fail>
+                            <failFast>false</failFast>
+                            <rules>
+                                <bannedDependencies>
+                                    <searchTransitive>true</searchTransitive>
+                                    <excludes>
+                                        <!-- We use log4j-over-slf4j, so we need to make sure we don't pull in
+                                             the actual log4j 1.x dependency to avoid conflicts. http://www.slf4j.org/legacy.html#log4j-over-slf4j -->
+                                        <exclude>log4j:log4j</exclude>
+                                        <!-- This dependency also pulls in log4j 1.x -->
+                                        <exclude>org.apache.log4j.wso2:log4j</exclude>
+                                        <!-- We use jcl-over-slf4j, so we need to make sure we don't pull in
+                                             the actual commons-logging dependency to avoid conflicts. http://www.slf4j.org/legacy.html#log4j-over-slf4j -->
```

Comment:
```suggestion
                                             the actual commons-logging dependency to avoid conflicts. http://www.slf4j.org/legacy.html#jclOverSLF4J -->
```